### ISSUE TITLE
Ubuntu 20.04 (focal): Depends: gir1.2-webkit-3.0 but it is not installable

### ISSUE THREAD

**Issue by LittleWhole** [gabrielbarbosanascimento/Up-Clock#23]
When trying to install up-clock on disco, eoan, or focal, the following error occurs:

```sh
The following packages have unmet dependencies:
 up-clock : Depends: python:any (>= 2.6.6-7~)
            Depends: gir1.2-webkit-3.0 but it is not installable
```

I believe the updated version of this package is `gir1.2-webkit2-4.0`, as `gir1.2-webkit-3.0` is no longer available on disco/eoan/focal. However, up-clock will not install even with `gir1.2-webkit2-4.0` installed, it requires `gir1.2-webkit-3.0`.

**Comment by apandada1** [comment]
@LittleWhole Thanks for reporting, and sorry for the delay. Ubuntu removed support for webkit v1 in favor of webkit2. I have started porting the code to `webkit2gtk` (already done) and python3 in [my fork](https://github.com/apandada1/up-clock). I will send a pull request to this repository once it is complete. Then I will build it in the Launchpad PPA.

**Comment by apandada1** [comment]
PPA package works.

**Comment by LittleWhole** [comment]
Thanks so much :D
